recipe/meta.yaml

```diff
@@ -206,17 +217,18 @@ outputs:
         - ninja
         # Keep libprotobuf here so that a compatibile version
         # of protobuf is installed between build and host
-        - libprotobuf
-        - protobuf
+        # TODO(baszalmstra): There are linker errors on Windows when using protobuf from conda-forge
+        - libprotobuf #[not win]
+        - protobuf    #[not win]
```

Comment:
You probably need to set the macro `PROTOBUF_USE_DLLS` (though hard to tell without more context).

Reply:
We have shared protobuf builds, which is not so often used elsewhere, hence many build scripts aren't prepared for that.

The way shared libraries work on windows (c.f. `__declspec(dll{im,ex}port)`), you need to ensure that the symbols are actually visible to the linker, and the macro I mentioned does that.

Reply:
Let me get back to this after I solved the current list of issues. :)